Public/Src/FrontEnd/UnitTests/Libs/Sdk.Transformers.ts

```diff
@@ -168,6 +168,9 @@ namespace Transformer {
             /** Set outputs to remain writable */
             keepOutputsWritable?: boolean;
 
+            /** Whether this process requires admin privilege */
+            requiresAdmin?: boolean;
+
 			/** Whether this process should run in an isolated container (i.e. filesystem isolation)

```

Comment:
>	 [](start = 1, length = 1)

tabs vs spaces

Reply:
I think mine is spaces (consistent with the previous fields). The container fields are all tabs.

---
In reply to: [276821383](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276821383) [](ancestors = 276821383)